README.md

```diff
@@ -10,6 +10,31 @@ Installation
 4. `sh ucsd-cs120-wi16-pa-tester/install.sh`
 5. `make test`
 
+## Usage
+#### Makefile
+the install.sh will echo the test target into the Makefile, 
+you can change the file name if necessary.
+
+2020-1-19: The mykernel2.c and mykernel2.h has been changed to mycode2.c and mycode2.h
```

Comment:
I don't think this change is necessary. After all, we're using version control. If people want to know what's changed, they can just look at the commit log :^)

Reply:
OK, I agree